Dockerfile

```diff
@@ -1,19 +1,11 @@
-FROM node:erbium-buster-slim as runner
-
-RUN apt-get update && \
-    apt-get install -y jq && \
-    apt-get purge --auto-remove -y && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+FROM node:14-alpine
+RUN apk add --no-cache jq coreutils bash
 
 ENV NO_UPDATE_NOTIFIER true
 
 WORKDIR /opt/test-runner
+COPY . .
 
-# Pre-install packages
-COPY package.json .
-COPY package-lock.json .
```

Comment:
Why did you remove this bit of the Dockerfile? This is supposed to make better use of Docker layer caching as those two files are less likely to change than the other files

Reply:
Since I think those get copied when doing COPY . .

Reply:
Yes, but the idea is that you copy just those files first, then do the `npm install` and after that do the `COPY . .`. This has the effect of docker caching the NPM install layer if these two files don't change. And as the NPM install step is relatively quite long, this can speed up builds.